internal/jobs/tiktok.go

```diff
@@ -260,6 +301,76 @@ func (ttt *TikTokTranscriber) ExecuteJob(j types.Job) (types.JobResult, error) {
 	return types.JobResult{Data: jsonData}, nil
 }
 
+// executeSearchByQuery runs the epctex/tiktok-search-scraper actor and returns results
+func (ttt *TikTokTranscriber) executeSearchByQuery(j types.Job, a *teeargs.TikTokSearchByQueryArguments) (types.JobResult, error) {
+	if ttt.configuration.ApifyApiKey == "" {
```

Comment:
Hmmmm..... let's see. Ignoring the possibility of bugs in the tee-indexer (which are not the miner's fault), we only get here if the miner has published the capability, meaning that we detected an Apify key on startup. This means that either the capability detection is wrong (again, not the miner's fault) or the miner somehow found a way to fool the system (which should probably be very harshly penalized).

All of this to say, should we perhaps have a separate stat for this sort of error, instead of lumping them together with the TiktokErrors? (This also applies below, and for Reddit). WDYT?

Reply:
Great point - for twitter, we accumulate in TwitterAuthErrors, which fire when there are issues creating a client for apify, api, or credential based clients - BUT, these only catch general errors in creation, not necessarily auth errors.

Removed and added to when the client is created, to at least follow similar patterns we have with other api-based  clients.  (this could also tie into use detecting when an actor is NOT rented by a particular key!)

Reply:
yeah I have a bunch of TODOs for that in the code, and also this: https://github.com/masa-finance/tee-indexer/issues/377